apps/academy/src/styles.css

```diff
@@ -26,7 +26,7 @@
 }
 
 .academy-grid .text-bttf-lg {
-  @apply text-bttf mb-6 text-center text-5xl lg:text-8xl;
+  @apply text-bttf mb-6 text-center text-5xl lg:text-7xl;
```

Comment:
I can see the positives with this: If I only focus on the text, it's easier on my eyes, and more homogeneous with the other uppercase texts further down in the page.
cons: it makes the pixel image very over-powering compared to the 'learn web3 with friends' text, so quite imbalanced.
Looking forward to some synchronous design sessions again after launch.

Reply:
i can see where this is coming from, and i suspect it differs by screen size too. my motivation for downsizing was the over-crowding of the logo + navbar + this text.